telco-examples/mgmt-cluster/README.md

```diff
@@ -0,0 +1,66 @@
+
+# Management Cluster
+
+This is an example of using Edge Image Builder (EIB) to generate a management cluster iso image for SUSE ATIP. The management cluster will contain the following components:
+- SUSE Linux Enterprise Micro 5.5 RT Kernel (SLE Micro RT)
```

Comment:
I would expect RT to only be used for the downstream/workload clusters, is this just for convenience to avoid downloading two images?

Reply:
yes, that was my original idea